enhancements/sandboxed-containers/sandboxed-containers-tech-preview.md

```diff
@@ -0,0 +1,620 @@
+                            name:
+                              description: Name of the failed node
+                              type: string
+                          required:
+                          - error
+                          - name
+                          type: object
+                        type: array
+                    type: object
+                  inProgress:
+                    description: InProgress reflects the status of nodes that are
+                      in the process of kata uninstallation
+                    properties:
+                      binariesUninstallNodesList:
+                        items:
+                          type: string
+                        type: array
+                      inProgressNodesCount:
+                        type: integer
+                    type: object
+                type: object
+              upgradeStatus:
+                description: Upgradestatus reflects the status of the ongoing kata
+                  upgrade
+                type: object
+            required:
+            - kataImage
+            - runtimeClass
+            - totalNodesCount
+            type: object
+        type: object
+    served: true
+    storage: true
+    subresources:
+      status: {}
+status:
+  acceptedNames:
+    kind: ""
+    plural: ""
+  conditions: []
+  storedVersions: []
+```
+
+When deploying the sandboxed containers operator, the admin has the option
+to deploy it on all its worker nodes, or to a subset of the worker nodes.
+
+The former is achieved by:
+```yaml=
+apiVersion: kataconfiguration.openshift.io/v1
+kind: KataConfig
+metadata:
+  name: example-kataconfig
+```
+
+And the latter can be done by taking advantage of the PoolSelector, and specifying the matching labels, as:
+```yaml=
+apiVersion: kataconfiguration.openshift.io/v1
+kind: KataConfig
+metadata:
+  name: example-kataconfig
+spec:
+  kataConfigPoolSelector:
+    matchLabels:
+       custom-kata1: test 
+```
+
+Thanks to the Kubernetes Runtime Class [scheduling
+support](https://v1-16.docs.kubernetes.io/docs/concepts/containers/runtime-class/#scheduling)
+present since `v1.16.0`, we ensure that when a workload requires a
+specific `RuntimeClass`, it will be scheduled on nodes that
+supports the specific `RuntimeClass`.
+
+Finally, it's important to mention that altough there's a **strong**
+preference on using CoreOS Extensions for actually installing Kata
+Containers, QEMU, and QEMU dependencies on the worker nodes, it still
+needs an approval from the development team.  A second option, which is
+still doable and should not damage the cluster in any possible way is
+simply copying the kata-containers binaries to the worker nodes, and relying
+on CoreOS Extensions only for QEMU, and QEMU dependencies.  Those options
+are extensively discussed as part of the next section.
+
+### Open Questions
+
+There's yet no consensus on how the sandboxed containers artifacts will be
```

Comment:
I would drop this and instead frame it as extensions as the proposed approach and copying as the alternative.